src/lib/get-is-beta-product.ts

```diff
@@ -1,14 +1,19 @@
 import { ProductSlug } from 'types/products'
+import { SectionOption } from './learn-client/types'
 
 /**
  * Given a `ProductSlug`, returns `true` if the associated Product is a beta
  * product and `false` otherwise. Intended for use in server-side code. If this
  * data is needed client-side, use the more optimized `useIsProductBeta` hook
  * provided by `AllProductDataProvider`.
  */
-const getIsBetaProduct = (productSlug: ProductSlug): boolean => {
+const getIsBetaProduct = (
+	productSlug: ProductSlug | SectionOption
```

Comment:
Since 'waf' isn't a product slug but it will be in the beta product slugs array, had to adjust this.